client/js/components/GoogleMapSearch/GoogleMapsSearch.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import { View } from "react-native";
+import { GooglePlacesAutocomplete } from "react-native-google-places-autocomplete";
+import styles from "./styles";
+import globalStyles from "../../config/globalStyles";
+
+class GooglePlacesInput extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <View style={styles.container}>
+        <GooglePlacesAutocomplete
+          placeholder="Where are you going?"
+          minLength={2}
+          autoFocus={false}
+          returnKeyType={"search"}
+          keyboardAppearance={"light"}
+          listViewDisplayed="auto"
+          fetchDetails={true}
+          renderDescription={row => row.description}
+          onPress={(data, details = null) => {
+            this.props.updateContextLocation(details.geometry.location);
+            console.log(details);
+          }}
+          getDefaultValue={() => ""}
```

Comment:
If you do not need this prop, remove it.

Reply:
I need this prop to say the default value is empty